temporal-serviceclient/src/main/java/io/temporal/serviceclient/SimpleSslContextBuilder.java

```diff
@@ -47,21 +47,23 @@
           // gRPC requires http2 protocol.
           ApplicationProtocolNames.HTTP_2);
 
-  private final InputStream keyCertChainInputStream;
-  private final InputStream keyInputStream;
+  private InputStream keyCertChain;
+  private InputStream key;
   private TrustManager trustManager;
   private boolean useInsecureTrustManager;
   private String keyPassword;
 
   /**
-   * @param keyCertChainInputStream - an input stream for an X.509 client certificate chain in PEM
-   *     format.
-   * @param keyInputStream - an input stream for a PKCS#8 client private key in PEM format.
+   * @param keyCertChain - an input stream for an X.509 client certificate chain in PEM format.
```

Comment:
Why is it keyCertChain? From the description is sounds like certChain to me. Or maybe certChainPem?

Reply:
Again, borrowed the name from the underlying SslContextBuilder.

Reply:
Same as above.